### DISCUSSION TITLE
Im waiting for 4 month, for my Student Education approvment

### DISCUSSION THREAD

**Discussion by itsNyos** [community/community#125449] [GitHub Education]
### Hi everyone,

Hi everyone,
Hello, I'm still waiting for my Education benefits.
I applied for it on January and i haven't got any mails, an even if i had, it wouldn't be possible for me anymore to see it 4 month ago.

What can i do?

This is written on the "Get your GitHub benefits" Website:

Approved
Submitted Jan 20, 2024

Hi, Philipp! You were last verified as a student on . It is not necessary for you to re verify at this time. You have an outstanding discount request for @itsNyos. We will get back to you soon.

**Comment by kazmatw** [comment]
Hello Philipp,

Thank you for reaching out and expressing your concerns regarding the delay in receiving your education benefits. I understand it can be frustrating to wait, especially without clear updates.

Since your application was approved on January 20, 2024, and there hasn't been any further communication, here are a few steps you can take:

Check Your Email Spam/Junk Folder: Sometimes, emails from new senders can be mistakenly filtered out. If you haven't already, please check these folders for any missed communications regarding your benefits.

Update Your Contact Information: Since you mentioned that it might not be possible for you to see emails from 4 months ago, make sure your contact information is up-to-date on the platform to receive future notifications properly.

Direct Inquiry: I recommend directly contacting the support or help desk associated with the GitHub benefits program. They can provide specific details regarding the status of your discount request for @itsNyos.

Documentation and Records: Keep a record of your application approval and any correspondence you might have related to this matter. This documentation can be helpful in expediting your inquiry.

If there’s anything more I can assist you with, please let me know. Hoping your issue gets resolved soon!
If you find this information useful, please consider selecting this as the answer to help others in the community identify effective solutions. Thank you!
